office.html

```diff
@@ -824,22 +823,22 @@ <h3 class="title">Harsh Kasyap</h3>
             <div class="link-section row col-lg-4 col-md-12 col-sm-12">
               <!--Grid column-->
               <div class="links col-lg-6 col-md-6 col-sm-6 col-xs-6">
-                <h6 class="text-uppercase">Important Links</h6>
+                <h6 class="text-uppercase">Club Links</h6>
```

Comment:
this  should not be club links. Change it back to Important links. Refer to master head for this